contracts/protocol/bases/SellerBase.sol

```diff
@@ -39,6 +41,34 @@ contract SellerBase is ProtocolBase, IBosonAccountEvents {
         // Cache protocol lookups for reference
         ProtocolLib.ProtocolLookups storage lookups = protocolLookups();
 
+        // Get message sender
+        address sender = msgSender();
+
+        // Do caller and uniqueness checks based on auth type
+        if (_seller.admin == address(0)) {
```

Comment:
Should this not be explicitly checking to ensure that the _authToken isn't null ?
something like : 
if (_seller.admin == address(0) && isNotNull(_authToken))  <-- or however best to do it given the Struct ?

Reply:
This is already checked in the `require` in line 46.